slack/welcome-message.txt

```diff
@@ -1,6 +1,7 @@
 We're thrilled to welcome you to the Keep :keep: community! 
 
 *Here's a quick overview of the available channels:*
+<#CD7NK42BV|keep_ru> - Channel for Keep's Russian-speaking community
```

Comment:
I think we should have this at the bottom of the list!

Reply:
Oh you're totally right!

Reply:
Actually it should be located right under #keep. Like it is in the Slack